### PR TITLE
Layout responsive, form container

### PR DESCRIPTION
**Pull request by Yiwahpsp**
fix: 
- layout responsive [add padding-x]
 
add: 
- tailwind config [border, font size]
- reset button
- form container (for wrapping all form)
- background

### CODE REVIEW

File: .env.example
Comment (shalluv) [inline .env.example:1]: don't delete this krub

File: src/components/form/form-container.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (shalluv) [inline src/components/form/form-container.tsx:1]: Please remove this line

File: src/components/form/form-container.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export function FormContainer({
+  children,
+  className,
+}: {
+  children: React.ReactNode
+  className?: string
+}) {
+  return (
+    <div
+      className={`flex w-full max-w-3xl flex-col gap-5 rounded-box border-default bg-white p-10 shadow-default ${className}`}
```
Comment (shalluv) [inline src/components/form/form-container.tsx:12]: Please utilize the `cn` function in `utils`

**Review by shalluv — CHANGES_REQUESTED** [review]

**Comment by shalluv** [comment]
Please help _take some screenshots or capture videos_ of the **form** page for me.

**Comment by Yiwahpsp** [comment]
https://github.com/user-attachments/assets/321416ca-1cf7-4875-8ec1-c1b8568acbe8

layout

**Review by shalluv — APPROVED** [review]

**Pull request merged**